.env.example

```diff
@@ -4,7 +4,7 @@ SENTRY_DSN=https://example.ingest.sentry.io/66666
 BTCV_NETWORK=bitcoinvaultTestnet
 
 #testnet.bitcoinvault.global(testnet), e1.electrumx.bitcoinvault.global(mainnet)
-HOST=testnet.bitcoinvault.global
+HOSTS=testnet.bitcoinvault.global
```

Comment:
You renamed it to `HOSTS` but there's one host anyway. What's the purpose here? Could you give an example of how it'd look like in the case of multiple hosts?

Reply:
HOSTS=e1.electrumx.bitcoinvault.global,e2.electrumx.bitcoinvault.global,e3.electrumx.bitcoinvault.global